### DISCUSSION TITLE
Error when trying to create a .exe file

### DISCUSSION THREAD

**Discussion by jBavanati** [TDAmeritrade/stumpy#1049] [Help: Coding & Implementations]
Hi
I created a simple python program and created a .exe file using `pyinstaller`, and this is the error I get when I try to open the .exe file:
![Screenshot_2024-12-06-21-22-10_12660](https://github.com/user-attachments/assets/6ec8a26c-2696-4825-ac19-91a51fbf9c12)

command I used for creating a .exe file: `pyinstaller --onefile --noconsole --name "program" script.py`
and I installed stumpy package using `python -m pip install stumpy`
It's worth noting that there is not any trace of `gpu stump` in my code.

**Comment by jBavanati** [comment]
Also the app loads fine when I directly open the program using `python script.py`

**Comment by jBavanati** [comment]
Update: I tried cloning the stumpy directory from this repository and then creating the .exe file using `pyinstaller --onefile --noconsole --name "file" --add-data "path_to_stumpy_directory;stumpy" script.py`. now I get this error:
![Screenshot_2024-12-06-22-59-48_9536](https://github.com/user-attachments/assets/02654bb1-e217-4547-8ec5-839c307332b9)

**Comment by patoalejor** [comment]
I have the same problem now, do you have any updates on this @jBavanati ?
